docs/api/native-image.md

```diff
@@ -137,15 +137,17 @@ let image = nativeImage.createFromPath('/Users/somebody/images/icon.png')
 console.log(image)
 ```
 
-### `nativeImage.createFromBuffer(buffer[, scaleFactor])`
+### `nativeImage.createFromBuffer(buffer[width, height, scaleFactor])`
 
 * `buffer` [Buffer][buffer]
+* `width` Integer (optional)
```

Comment:
I already wrote this but if you take a look at line 411, it seems to be backwards compatible.